### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by Patchescc** [mirthAI/Fast-DDPM#9]
I noticed that the learning rate in the code is set to 0.00002 (2×10^-5), while the paper indicates a learning rate of 2×10^-4. I'm unsure which one is correct

**Comment by Patchescc** [comment]
Hello!

I am running the codes, and after 60,000 iterations, I observed that the loss is decreasing but not reaching a very low value. It still ranges between 8 and 200. I might have made an error during the preloading process. Could you share the expected loss ranges or guide me on the correct values?

**Comment by JunBond** [comment]
> Hello again!
> 
> I am running the codes, and after 60,000 iterations, I observed that the loss is decreasing but not reaching a very low value. It still ranges between 8 and 200. I might have made an error during the preloading process. Could you share the expected loss ranges or guide me on the correct values?

So did you slove the problem? I also find the problem of the learning rate, and the loss did not decrease, the dataset I run was PMUB.

**Comment by wyb-sudo** [comment]
你好，请问学习率，您知道设置多少了吗？是按照论文走吗？另外我想请问你是否也将预训练权重加载上了，想请问您大概设置了多少epoch。感谢回答！！！

**Comment by JunBond** [comment]
> 你好，请问学习率，您知道设置多少了吗？是按照论文走吗？另外我想请问你是否也将预训练权重加载上了，想请问您大概设置了多少epoch。感谢回答！！！

I prefer the learning rate of 0.0002 in the paper, and the epoch remains unchanged according to the original configuration file, but the loss cannot be reduced. I want to load the pre-trained weights, but the pre_trained interface in the code seems to have some problems. I am also studying this, and hope the author can update it

**Comment by Day-Dreamland** [comment]
您好，我想问一下现在可以重新加载预训练权重文件了吗？我发现按照给的采样指令运行，一直会出现”时间步999超出beta数组长度10“的问题
